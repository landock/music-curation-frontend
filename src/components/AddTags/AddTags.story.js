import React from 'react';
import { storiesOf } from '@storybook/react';
import { action } from '@storybook/addon-actions';
import { linkTo } from '@storybook/addon-links';
import AddTags from '.';

storiesOf('AddTags', module).add('default', () => (
  <AddTags
    tags={[{ id: 1, title: 'Breakfast' }, { id: 2, title: 'Native American' }]}
  />
));
