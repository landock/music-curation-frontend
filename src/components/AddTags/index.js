import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';
import TagCollection from '../TagCollection';
class AddTags extends PureComponent {
  render() {
    return (
      <div>
        <p>Add tags</p>
        <TagCollection tags={this.props.tags} />
      </div>
    );
  }
}

AddTags.propTypes = {
  tags: PropTypes.array,
};

export default AddTags;
