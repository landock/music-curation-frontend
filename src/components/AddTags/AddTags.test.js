
import React from 'react'
import { shallow, mount, render } from 'enzyme';
import renderer from 'react-test-renderer'
import AddTags from '.'

it('AddTags: default', () => {
  const component = shallow(<AddTags />)
  expect(component.text()).toEqual('<Header />Add Tags<TagCollection />')
})
