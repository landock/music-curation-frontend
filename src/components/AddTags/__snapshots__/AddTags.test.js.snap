// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddTags: default 1`] = `
<div>
  <h5
    className="ui header"
  >
    Tags
  </h5>
  <div>
    <p>
      Add Tags
    </p>
    <span>
      No tags
    </span>
  </div>
</div>
`;
